client/src/elm/Backend/Person/Form.elm

```diff
@@ -332,7 +332,7 @@ applyDefaultValues currentDate maybeVillage isChw maybeRelatedPerson operation f
                 |> applyDefaultTextInput firstName defaultFirstName identity
                 |> applyDefaultTextInput secondName defaultSecondName identity
                 |> applyDefaultTextInput nationalIdNumber defaultNationalIdNumber identity
-                |> applyDefaultTextInput birthDate defaultBirthDate formatYYYYMMDD
+                |> applyDefaultTextInput birthDate defaultBirthDate formatDDMMyyyy
```

Comment:
This is data we send to backend.
All that is sent to backend is at `YYYYMMDD` format.
Revert.